[2022-12-04 11:04:14,109] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-04T00:30:00+00:00 [queued]>
[2022-12-04 11:04:14,924] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-04T00:30:00+00:00 [queued]>
[2022-12-04 11:04:14,925] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 11:04:14,929] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-04 11:04:14,930] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 11:04:15,012] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-11-04 00:30:00+00:00
[2022-12-04 11:04:15,065] {standard_task_runner.py:52} INFO - Started process 12264 to run task
[2022-12-04 11:04:15,131] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'local_to_gcs_task', 'scheduled__2022-11-04T00:30:00+00:00', '--job-id', '5134', '--raw', '--subdir', 'DAGS_FOLDER/AirflowClean2.py', '--cfg-path', '/tmp/tmpj4ip84is', '--error-file', '/tmp/tmpcq9uxc9p']
[2022-12-04 11:04:15,140] {standard_task_runner.py:77} INFO - Job 5134: Subtask local_to_gcs_task
[2022-12-04 11:04:15,806] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-04T00:30:00+00:00 [running]> on host 9e44869ff3ef
[2022-12-04 11:04:16,173] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Fadlil
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-04T00:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-04T00:30:00+00:00
[2022-12-04 11:04:17,028] {python.py:175} INFO - Done. Returned value was: None
[2022-12-04 11:04:17,305] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=local_to_gcs_task, execution_date=20221104T003000, start_date=20221204T110414, end_date=20221204T110417
[2022-12-04 11:04:17,523] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-04 11:04:17,700] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
